transport/transport-api/src/main/java/com/google/android/datatransport/Destination.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.android.datatransport;
+
+import androidx.annotation.Nullable;
+
+public interface Destination {
```

Comment:
`Destination` seems to be only relevant to `transport-runtime`, while for `transport-api` it is not very meaningful as it only cares about `Transports` and `TransportFactories` and does not know/care how any particular factory is implemented and if they have any destinations at all, also name and extras seem to be very specific to `transport-runtime` implementation. That said, can you move this interface into `transport-runtime`?